openshift-ci/Dockerfile.bundle.ci

```diff
@@ -2,14 +2,14 @@
 FROM scratch
 
 ARG CHANNEL=4.6
-ARG ZVERSION=0
+ARG ZVERSION=1
 
 LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=performance-addon-operator
-LABEL operators.operatorframework.io.bundle.channels.v1=4.6.0
-LABEL operators.operatorframework.io.bundle.channel.default.v1=4.6.0
+LABEL operators.operatorframework.io.bundle.channels.v1=4.6.1
```

Comment:
Are you sure about this? If this is related to the Subscription channel, then it needs to stay the same (4.6.0).

Reply:
ill top that with another question, should'nt this match the content of `annotations.yaml` ?
e.g.
```
  operators.operatorframework.io.bundle.channels.v1: "4.6"
  operators.operatorframework.io.bundle.channel.default.v1: "4.6"
```
(as shown in this patch set)

Reply:
yes, here we drifted away. We should not have included the z-version in the channel name, which is a mistake dating back probably to 4.5 or earlier. It didn't break because we're not yet on operatorhub, so most likely noones is attempting upgrades.

It should most likely match `annotations.json` indeed, I'll check.

Reply:
ok, best we can do now is make the state not more broken than before, so I'll revert to 4.6.0 here. Then we will need to clean up the u/s channels state